### PR TITLE
Fix app-dir/next-font for Turbopack

### PR DESCRIPTION
**Pull request by timneutkens**
## What?

Updates the test checks for Turbopack.

Closes NEXT-2364

### CODE REVIEW

**Comment by timneutkens** [comment]
* `canary`
  * **#61678** <a href="https://app.graphite.dev/github/pr/vercel/next.js/61678?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @timneutkens and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: test/e2e/app-dir/next-font/next-font.test.ts [hunk 1]
```diff
@@ -52,8 +45,12 @@ describe('app dir - next/font', () => {
 
               // layout
               expect(JSON.parse($('#root-layout').text())).toEqual({
-                className: expect.stringMatching(/^__className_.{6}$/),
```
Comment (ForsakenHarmony) [inline test/e2e/app-dir/next-font/next-font.test.ts:55]: do we not want to keep the more specific regex for webpack?

**Review by ForsakenHarmony — APPROVED** [review]

File: test/e2e/app-dir/next-font/next-font.test.ts (hunk 1 above)
Comment (timneutkens) [reply test/e2e/app-dir/next-font/next-font.test.ts:55]: Tried to match them a bit more, overall the format really doesn't matter as long as the class is there

**Review by ForsakenHarmony — CHANGES_REQUESTED** [review]
actually it would be ideal to update the test manifest with this PR so the changes are tested before merging

**Pull request merged**